.pre-commit-config.yaml

```
@@ -8,5 +8,14 @@ repos:
   - repo: https://gitlab.com/pycqa/flake8
     rev: 3.7.9
     hooks:
-    - id: flake8
-      exclude: radis/io/hapi.py
\ No newline at end of file
+      - id: flake8
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.3.0
+    hooks:
+      - id: trailing-whitespace
+        exclude: |
+            (?x)(
+                ^.+.spec|
+                ^.+.dat
```

Hello ! I also excluded the .levels files, which are some other plain text input files. 

```suggestion
                ^.+.dat|
                ^.+.levels
```